module/Api.py

```diff
@@ -805,7 +806,11 @@ def getCaptchaTask(self, exclusive=False):
         if task:
             task.setWatingForUser(exclusive=exclusive)
             data, type, result = task.getCaptcha()
-            t = CaptchaTask(int(task.id), standard_b64encode(data), type, result)
+            if task.isInteractive() == 1:
+                data = json.dumps(data) #data = {url:..., sitekey:...}
```

Comment:
inline comments in python should have at least to spaces before the hash sign and one afterwards for readability.
e.g. `data = json.dumps(data)  # data = {url:..., sitekey:...}`